tests/integration/test_logcollector/test_age/test_age_basic.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+import os
+import sys
+import tempfile
+
+import pytest
+import wazuh_testing.logcollector as logcollector
+from wazuh_testing.tools import LOG_FILE_PATH
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools.monitoring import LOG_COLLECTOR_DETECTOR_PREFIX, AGENT_DETECTOR_PREFIX
+from wazuh_testing.tools.time import time_to_seconds, time
+
+# Marks
+pytestmark = pytest.mark.tier(level=0)
+
+# Configuration
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+configurations_path = os.path.join(test_data_path, 'wazuh_age.yaml')
+
+folder_path = os.path.join(tempfile.gettempdir(), 'wazuh_testing_age')
+folder_path_regex = os.path.join(folder_path, '*')
+
+local_internal_options = {'logcollector.vcheck_files': 0}
+
+if sys.platform == 'win32':
+    prefix = AGENT_DETECTOR_PREFIX
+else:
+    prefix = LOG_COLLECTOR_DETECTOR_PREFIX
```

Comment:
This was changed in the master branch. You will need to updated with the latest changes.